Makefile

```diff
@@ -6,6 +6,12 @@ IMAGE        = $(REGISTRY)machine-api-operator
 BUILD_IMAGE ?= registry.ci.openshift.org/openshift/release:golang-1.18
 GOLANGCI_LINT = go run ./vendor/github.com/golangci/golangci-lint/cmd/golangci-lint
 
+# ENVTEST_K8S_VERSION refers to the version of kubebuilder assets to be downloaded by envtest binary.
+ENVTEST_K8S_VERSION = 1.25
+
+PROJECT_DIR := $(shell dirname $(abspath $(lastword $(MAKEFILE_LIST))))
+ENVTEST = go run ${PROJECT_DIR}/vendor/sigs.k8s.io/controller-runtime/tools/setup-envtest
```

Comment:
These are duplicated below

Reply:
Do we want to have `1.26` version here as well?

Reply:
Yeah I think lets try getting all of the envtest updated to 1.26, and other deps if they need to as well

Reply:
This is still duplicating existing vars below, can you please double check what you've added here in a wider context